src/pkg/backup/details/details.go

```diff
@@ -516,6 +516,7 @@ type ExchangeInfo struct {
 	ItemType    ItemType  `json:"itemType,omitempty"`
 	Sender      string    `json:"sender,omitempty"`
 	Subject     string    `json:"subject,omitempty"`
+	Recipient   string    `json:"recipient,omitempty"`
```

Comment:
Per my other comment:
```suggestion
	Recipient   []string    `json:"recipient,omitempty"`
```

Reply:
Resolved within [1e09b2e](https://github.com/alcionai/corso/pull/2689/commits/1e09b2ebd5146ad4fde493e8c0bed702a80ea314)